src/controller/AutoCommissioner.cpp

```diff
@@ -499,6 +497,8 @@ CommissioningStage AutoCommissioner::GetNextCommissioningStageInternal(Commissio
         }
         return CommissioningStage::kFindOperational;
     case CommissioningStage::kFindOperational:
+        return CommissioningStage::kICDSendStayActive;
+    case CommissioningStage::kICDSendStayActive:
```

Comment:
Per previous discussions, maybe we can consider send the two commands in "SendComplete" stage, after we supports multiple commands in single invoke, we can merge it into a single command invoke transaction.

Reply:
updated, thanks